Enchancement/53 table styles

ryansbuendorf opened the pull request.

ryansbuendorf committed:
updated styles on tables to match new color theme

bonezee117 committed:
Various CSS changes

ryansbuendorf committed:
Merge branch 'dev' into enchancement/53_table_styles

ryansbuendorf committed:
fixed merge conflicts and shop nav bar

steven11sjf commented:
I'm having an issue where moving the mouse from a dropdown to its elements causes the elements to disappear unless you flick the cursor really fast. could you look into this?

steven11sjf committed:
Dropdown fixes

- adjusted margin on dropdown contents to remove empty space
- reduced padding on dropdown elements to make teams visible without
  scrolling
- pushed a blog entry I forgot about... oops

steven11sjf committed:
Merge branch 'dev' into enchancement/53_table_styles

- fix merge conflicts from deleting newstyles.css
